File: example/javalib/testing/6-test-group-stealing/foo/test/src/foo/GroupY2.java
package foo;

import org.junit.Test;

public class GroupY2 extends RandomTestsUtils {
  @Test
  public void test1() throws Exception {
    testGreeting("Odin", 34);
  }

  @Test
  public void test2() throws Exception {
    testGreeting("Pluto", 32);
  }

  @Test
  public void test3() throws Exception {
    testGreeting("Quetzal", 33);
  }
}

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupY2.kt
package foo

import org.junit.jupiter.api.Test

class GroupY2 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Odin", 34)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Pluto", 32)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Quetzal", 33)
    }
} 
